### PR TITLE
fix: remove perpetual spinner

### PR DESCRIPTION
**Pull request by hkang1**
## Description

Fix the problem of a permanent spinner showing on the model version header section.

## Test Plan

- [ ] navigate to any model VERSION page
- [ ] check that the page header does not have a permanent spinner overlay the page header section

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket

WEB-1065

### CODE REVIEW

File: webui/react/src/pages/ModelVersionDetails/ModelVersionHeader.tsx [hunk 1]
```diff
@@ -277,9 +277,7 @@ my_model.load_state_dict(ckpt['models_state_dict'][0])`;
             </Dropdown>
           </div>
         </div>
-        <Spinner spinning>
-          <InfoBox rows={infoRows} separator={false} />
-        </Spinner>
+        <InfoBox rows={infoRows} separator={false} />
```
Comment (hkang1) [inline webui/react/src/pages/ModelVersionDetails/ModelVersionHeader.tsx:280]: infoRows is only dependent on `users` and `modelVersion`. `modelVersion` will always be defined where as `users` is a `Loadable` state, but it already has it's own local spinner. So it's ok to go ahead and always render `Infobox`.

**Review by keita-determined — APPROVED** [review]
LGTM
